java/ql/src/experimental/Security/CWE/CWE-489/devMode.ql

```diff
@@ -0,0 +1,16 @@
+import java
+import experimental.semmle.code.xml.StrutsXML
+
+bindingset[path]
+predicate isBadPath(string path) {
+  path.matches("%example%") or
+  path.matches("%test%") or
+  path.matches("%demo%")
```

Comment:
`isBadPath` is a little bit unspecific, maybe it would be better to name it `isNonProductionPath`.

Additionally you can use a [set literal expression](https://help.semmle.com/QL/ql-handbook/expressions.html#set-literal-expressions) to only have one `path.matches(...)`:
```ql
path.matches(["%example%", "%test%", "%demo%"])
```

Reply:
Suggest using case-insensitive regex matching also. Example of syntax: `result.regexpMatch("(?i).*(selfCert|selfSign|validat|verif|trust).*")`

Reply:
done :+1: